src/components/encoding-pane/range-filter-shelf.scss

```diff
@@ -0,0 +1,324 @@
+.range-input {
+  display: flex;
+  justify-content: space-between;
+
+  input {
+    width: 5;
+  }
+
+  span {
+    margin: 2px;
+  }
+}
+
+.range-filter-pane {
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+:global {
```

Comment:
For some reasons the styles in package rc-slider were not working. I had to copy paste the css file it uses.